modules/replication/module.py

```diff
@@ -730,6 +745,12 @@ def renew_reqs(self, processors_set):
             if req in reqs_need_pre_prep:
                 # current view is equal to self.id since we are primary
                 req[REQUEST].set_view(self.id)
+                # Increment sequence number and assign
+                self.rep[self.id].inc_seq_num()
```

Comment:
Does this miss one seq num if there has been a reset of seq_num up around lines 630-640? Maybe we should always inc seq num after assigning it to a req instead?